dotcom-rendering/src/web/components/SignInGate/gates/main-variant.tsx

```diff
@@ -15,19 +15,57 @@ const SignInGateMain = React.lazy(() => {
 	});
 });
 
+const SignInGateMainCheckoutComplete = React.lazy(() => {
+	const { start, end } = initPerf('SignInGateMainCheckoutComplete');
+	start();
+	return import(
+		/* webpackChunkName: "SignInGateMain" */ '../gateDesigns/SignInGateMainCheckoutComplete'
+	).then((module) => {
+		end();
+		return { default: module.SignInGateMainCheckoutComplete };
+	});
+});
+
+// TODO investigate data tracking component name requirements?
+// TODO Add context Comments
+const showCheckoutCompletePersonalisation = true;
```

Comment:
Not sure what you meant by this here?

Reply:
We need to address the tracking that is being send to Ophan. We need to differentiate the different gate component desings that will be possible within the main-variant.

Reply:
Ah I see. 

Was a bit confused about the explicit setting of the boolean varaible, but I see that this is being used here as a switch. 

My thoughts on this are if it needs a code change and a redeploy to turn off the component we might as well just delete the code since it wouldn't be that much extra work to back out the change and we'd have the code in the history. If we're able to control the displaying of the component with a switch that's external to the code base, then it makes sense to have a flag

Reply:
Added a flag